_partials/integrations/hibernate-ogm.html.haml

```diff
@@ -0,0 +1,3 @@
+The Hibernate team maintains the
```

Comment:
Where will this text be displayed? Does it matter that the Hibernate team maintains it? Wjat does integrate means?